docs/en/advanced-concepts/image-elements.md

```diff
@@ -34,7 +34,8 @@ To access these settings, you should use the Appium [Settings API](/docs/en/adva
 |fixImageTemplateSize|OpenCV will not allow the matching of a reference image / template if that image is larger than the base image to match against. It can happen that the reference image you send in has dimensions which are larger than the screenshot Appium retrieves. In this case the match will automatically fail. If you set this setting to `true`, Appium will resize the template to ensure it is at least smaller than the size of the screenshot.|`true` or `false`|`false`|
-|fixImageTemplateScale| Appium resizes a base image to fit its window size before matching them with OpenCV. If you set this setting to `true`, Appium scales a reference image you send in as the same scale Appium scales the base image to fit the window size. e.g. iOS screenshot is `750 × 1334` pixels base image. The window size is `375 x 667`. Appium rescale the base image to window size scaling it with `0.5`. A reference image is based on the screenshot size, never the image matches with the window size scale. This settings allow Appium to scale the reference image with `0.5`. [appium/appium-base-driver#306](https://github.com/appium/appium-base-driver/pull/306)| `true` or `false` | `false` |
+|fixImageTemplateScale| Appium resizes a base image to fit its window size before matching them with OpenCV. If you set this setting to `true`, Appium scales a reference image you send in as the same scale Appium scales the base image to fit the window size. e.g. iOS screenshot is `750 × 1334` pixels base image. The window size is `375 x 667`. Appium rescale the base image to window size scaling it with `0.5`. A reference image is based on the screenshot size, never the image matches with the window size scale. This settings allow Appium to scale the reference image with `0.5`. [appium-base-driver#306](https://github.com/appium/appium-base-driver/pull/306)| `true` or `false` | `false` |
+|defaultImageTemplateScale| Appium resizes a template image with arbitrary scale by default if you set this setting to `4.0`, for example. Defaults to `1.0` (Does not scale). Storing scaled template images might help decreasing your storage size than storing original images. e.g. A user has `270 × 32 pixels` template image oridinally `1080 × 126 pixels`. The user stores the former image in his storage since he would like to minimise storage size. If Appium expands it to fourth, the template image can be almost the same as the original one. OpenCV can identify the scaled template image as the originally one. This setting helps the case. [appium-base-driver#307](https://github.com/appium/appium-base-driver/pull/307)|`0.5`, `10.0`, `100`| `1.0` |
```

Comment:
Appium does not resize template images by default (the value of `1.0`). Although, storing scaled template images might help to save size of the storage. E.g. one has could represent `1080 × 126 pixels` area by `270 × 32 pixels` template image (the value of `defaultImageTemplateScale ` is expected to be set to `4.0`). Check [appium-base-driver#307](https://github.com/appium/appium-base-driver/pull/307) for more details.